Chatkit/Redux/Reducers/Model/RoomListReducer.swift

```diff
@@ -0,0 +1,145 @@
+
+extension Reducer.Model {
+    
+    struct RoomList: Reducing {
+        
+        // MARK: - Types
+        
+        typealias ActionType = Action
+        typealias StateType = RoomListState
+        typealias DependenciesType = NoDependencies
+        
+        // MARK: - Reducer
+        
+        static func reduce(action: ActionType, state: StateType, dependencies: DependenciesType) -> StateType {
+            if let action = action as? InitialStateAction {
+                return self.reduce(action: action, state: state, dependencies: dependencies)
```

Comment:
`self.` is redundant here (and other calls to `reduce` below)

Reply:
The usage of an implicit/explicit `self` is a matter o personal preference.

Reply:
OK cool, I have no issue if you like to use `self.`.  Just wasn't sure if it was intentional or not.